HealthcareSystem/GraphicalEditorServerTests/DataFactory/CreateEquipmentType.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Backend.Model.Manager;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GraphicalEditorServerTests.DataFactory
+{
+    public class CreateEquipmentType : ICreateTestObjectFactory<EquipmentType>
+    {
+        public EquipmentType CreateInvalidTestObject()
+        {
+            return new EquipmentType("fsdafafas",false);
+        }
+
+        public EquipmentType CreateValidTestObject()
+        {
+            return new EquipmentType("table",true);
+        }
+
+        public EquipmentType CreateValidTestObject(int id)
+        {
+            return new EquipmentType("table", true);
+        }
```

Comment:
You can see that the interface is too restrictive here already, it makes no sense to send an int here, you don't need it.